### ISSUE TITLE
Questions about region of force calculation 

### ISSUE THREAD

**Issue by hope366** [yzyray/lizzieyzy#21]
![無題](https://user-images.githubusercontent.com/63999713/141101726-3a1e88de-6021-42f4-bdbb-9f4c7ee3ae3d.png)
I see three items in the pull-down menu like this, but I couldn't see the difference between the three. 
Whichever of the three settings you choose, they look the same. What does the third "3" mean? "Keep this function only for three moves," I interpreted it this way, but it doesn't seem to be the case ...

**Comment by yzyray** [comment]
The 3 means in candidate's variations,after 3 moves the limit region will be invalid and the variations can show a stone out of the limited region by then.

**Comment by hope366** [comment]
"Set region of force culculation" has been set to "Custom moves: 3". In the figure below, only one stone is placed on the board, and the situation of the second analysis is shown, but the variation is displayed outside the specified range. 
<img src="https://user-images.githubusercontent.com/63999713/141225587-a329bf1e-3e4b-49b4-bee2-f6324291c47b.png" width="60%">
I also limited the analysis range and then placed 5 stones on the board, but the restricted area does not appear to be disabled. 
<img src="https://user-images.githubusercontent.com/63999713/141227189-663f2f5d-1609-4354-b6b5-d5fcc7e4c34a.png" width="60%">

**Comment by yzyray** [comment]
The `3` means in every candidate's variations,first three moves must be in the region.
In your first picture,the first 6 moves is in the region,so it is fine.

**Comment by yzyray** [comment]
I think the translation is not very proper,changed to `limit variation moves`

**Comment by hope366** [comment]
Thanks to you, I seem to understand what you are saying.
By the way, see the screenshot below. Despite being set to "3", the second movement of the variation is outside the limited area ... 
<img src="https://user-images.githubusercontent.com/63999713/141248567-6c6fa3a0-3f79-458c-a972-770aa20f16ca.jpg" width="60%">

**Comment by yzyray** [comment]
This no.9 candidates might be copied from another candidates Q5 due to symmetry,the original candidates Q5's variation move 2 should be in the limit region.

**Comment by hope366** [comment]
Although the amount of analysis is different, we were able to display the same variation. The second move of the original candidate Q5 was, as you say, within the restricted area.
Thanks to your wonderful explanation I was able to understand all the probrems on this issue. Thank you very much. 
<img src="https://user-images.githubusercontent.com/63999713/141268965-7b926448-71a0-4a42-bfe5-9b279ca81a23.png" width="60%">
